src/dev/flang/be/c/CExpr.java

```diff
@@ -88,7 +88,7 @@ static CExpr dummy(String text)
    *
    * @return the resulting expression
    */
-  static CExpr ident(String name)
+  static CIdent ident(String name)
```

Comment:
I do not agree with this. The idea ist that this creates an Expression, which is also documented. The result is of type CExpr, but might be some subtype of that, e.g., we might add comments or debugging code to every single expression produced.